### PR TITLE
Merge sample app into plugin repo, and upgrading to RN 0.60.6

### PR DESCRIPTION
**Pull request by Apekka**
The sample app source code is contained in the directory "example" in the repo.
First run `npm install`
To run it, you place yourself into the root repo (react-native-acc), and run `react-native run-android --root example/` after the build and the install, run `react-native start`

Using Android SDK 4.0.4.

======

Major edits :

- All the modules containing the methods (RNAccDeviceInfoModule, RNAccTrackingModule...) were merged in only one module (This was mandatory for Android, because since RN0.60, there is autolinking, and it links ONE module), **RNAcc.h** for iOS and **RNAccModule.java** for Android. Their methods are called in **acc.js** (in the lib folder), this is the most important file. 

- A lot of changes were done in the sample app, because for example you must refactor your calls to the Acc API, like **Acc.push.setLocked** became **Acc.setPushLocked** 

- The sample app was merged into this repo, to simplify support and maintenance, so there was some changes in the build.gradle and podspec, so the app can retrieve easily our library.

- The SDK versions were updated so there is some changes in the build.gradle

### CODE REVIEW

File: example/App.js [hunk 1]
```diff
@@ -0,0 +1,244 @@
...
+//                 Acc.dismissView(prevScreen);
+//               }
+//               if (prevScreen === "View2") {
+//                 Acc.dismissView(prevScreen);
+//               }
+//             }
+//         }}/>;
+//   }
+
+// }
```
Comment (Ulrico972) [inline example/App.js]: You could remove this.

**Comment by oristanovic** [comment]
👍 We just need to have a migration guide covering these integration changes.

File: example/app/components/analytics/DeviceTagScreen.js [hunk 2]
```diff
@@ -16,7 +16,6 @@ import {
 import Button from 'react-native-button';
 import Acc from 'react-native-acc';
```
[the remark this answers is not in the document]
Comment (Apekka) [reply example/app/components/analytics/DeviceTagScreen.js]: OK !

File: example/App.js (hunk 1 above)
Comment (Apekka) [reply example/App.js]: Done

**Comment by mikaoelitiana** [comment]
Hello. Do you have an idea when this would be merged? Some of our apps are waiting it for RN upgrade.

**Comment by oristanovic** [comment]
Hello, we're be releasing this in a couple of weeks time. Sorry for the delay.

**Pull request merged**